Week-2/poem-webpage.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Poem Webpage</title>
+</head>
+<body>
+    <p>
+        I'm Nobody! Who are you? <br>
```

Comment:
Can you work on this to make it block elements instead of br

Reply:
I have worked on this already. Please check week2 PR.